docs/development/application-logic.md

```diff
@@ -245,6 +245,9 @@ transit processor (Littlepay).
 
     [`cal-itp/littlepay`][littlepay]
 
+
+### Littlepay
```

Comment:
Can we please move this heading _above_ the `littlepay` links? So they are part of this section?

And also update the entrypoint link to littlepay views (if that makes sense)